commit 54a29913c5de797da6adb69e03b38487fef451b4
Author: Fabio M. Di Nitto <fdinitto@redhat.com>
Date:   Mon Apr 2 13:47:41 2012 +0200

    man: update fenced.8 to reflect a limitation in XML/DTD implementation
    
    Resolves: rhbz#808441
    
    Reviewed-by: Christine Caulfield <ccaulfie@redhat.com>
    Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>

diff --git a/fence/man/fenced.8 b/fence/man/fenced.8
index 64f26e0..b172151 100644
--- a/fence/man/fenced.8
+++ b/fence/man/fenced.8
@@ -335,6 +335,11 @@ power never being fully disabled to the node.
 </clusternode>
 .fi
 
+.SS NOTES
+
+Due to a limitation in XML/DTD validation, the name="" value within
+the device or fencedevice section cannot start with a number.
+
 .SS Hardware-specific settings
 
 Find documentation for configuring specific devices from the device
